.github/CODEOWNERS

```diff
@@ -826,6 +826,7 @@ packages/kbn-es @elastic/kibana-operations
 packages/kbn-es-archiver @elastic/kibana-operations
 packages/kbn-es-errors @elastic/kibana-core
 packages/kbn-es-query @elastic/kibana-app-services
+packages/kbn-es-types @elastic/kibana-core
```

Comment:
Mind adding APM UI team here (or me, I know you have a preference for the former though)

Reply:
This part of the codeowners file is automatically generated depending on the `owner` field of the `kibana.jsonc` file of the package.

AFAIK atm we can't have multiple teams owning a given package. @spalger mind confirming that?

If that's fine with you, I'll be pleased to switch ownership to the apm-ui team though, given Core didn't really contribute on those types at all IIRC. What do you think?

Reply:
Every package has a single owner, but additional entries can be added at the beginning of the codeowners file to support multiple teams reviewing changes to specific packages.

Reply:
I thought entries / lines in the codeowners files were not additive, and that the last entry in order for a given path was the one effectively applied.

Do you mean that adding a line with

```
packages/kbn-es-types @elastic/kibana-core @elastic/apm-ui
```

**before** L829 in the codeowners file would work here?

Reply:
> If that's fine with you, I'll be pleased to switch ownership to the apm-ui team though, given Core didn't really contribute on those types at all IIRC. What do you think?

perfectly fine with me :)

Reply:
Oof, you're right @pgayvallet... I need to fix that asap then

Reply:
Alright, with https://github.com/elastic/kibana/pull/140740 the `owner` property in `kibana.jsonc` can now be an array and include `@elastic/apm-ui` if you would like.

Reply:
@dgieselaar  Thanks to https://github.com/elastic/kibana/pull/140740, done in https://github.com/elastic/kibana/pull/140522/commits/0ff44b7f5c123e21a6fa4ccfc1086de0f3b5a10c

Reply:
I don't see it in the CODEOWNERS file? Should it be there?